DESCRIPTION

```diff
@@ -1,16 +1,21 @@
-Package: rsconnect
 Type: Package
-Title: Deployment Interface for R Markdown Documents and Shiny Applications
+Package: rsconnect
+Title: Deployment Interface for R Markdown Documents and Shiny
+    Applications
 Version: 0.8.29-1
 Authors@R: c(
-    person("Aron", "Atkins", role = c("aut", "cre"), email = "aron@posit.co"),
-    person("Jonathan", "McPherson", role = "aut", email = "jonathan@posit.co"),
+    person("Aron", "Atkins", , "aron@posit.co", role = c("aut", "cre")),
+    person("Jonathan", "McPherson", , "jonathan@posit.co", role = "aut"),
```

Comment:
how do you feel about expanding this set? when is appropriate?

Reply:
Happy to expand it 😄 This the general tidyverse policy on authors: https://github.com/tidyverse/tidyups/blob/main/004-governance.md#authors